examples/codecommit/trigger.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: codecommit.aws.upbound.io/v1beta1
+kind: Repository
+metadata:
+  name: example
+  annotations:
+    meta.upbound.io/example-id: codecommit/v1beta1/repository
+  labels:
+    testing.upbound.io/example-name: example
+spec:
+  forProvider:
+    description: This is the Sample App Repository
+    region: us-west-1
+---
+apiVersion: sns.aws.upbound.io/v1beta1
+kind: Topic
+metadata:
+  name: example
+  labels:
+    testing.upbound.io/example-name: example
+spec:
+  forProvider:
+    region: us-west-1
+---
+apiVersion: codecommit.aws.upbound.io/v1beta1
+kind: Trigger
+metadata:
+  name: example
+  annotations:
+    meta.upbound.io/example-id: codecommit/v1beta1/trigger
+    uptest.upbound.io/post-assert-hook: testhooks/delete-trigger.sh    
```

Comment:
It looks like this is causing problems when both examples passed together.
Consider using `upjet.upbound.io/manual-intervention` annotation.

Reply:
We just introduced support for pre/post delete hooks with [uptest v0.2.0](https://github.com/upbound/uptest/releases/tag/v0.2.0). Please consider them to fix the problem mentioned above.

Reply:
Done.